Dockerfile

```
@@ -23,25 +23,27 @@ RUN swift package resolve \
 # Copy entire repo into container
 COPY . .
 
+RUN mkdir /staging
+
 # Build the application, with optimizations, with static linking, and using jemalloc
 # N.B.: The static version of jemalloc is incompatible with the static Swift runtime.
-RUN swift build -c release \
+RUN --mount=type=cache,target=/build/.build \
+    swift build -c release \
         --product {{name}} \
         --static-swift-stdlib \
-        -Xlinker -ljemalloc
+        -Xlinker -ljemalloc && \
+    # Copy main executable to staging area
+    cp "$(swift build --package-path /build -c release --show-bin-path)/{{name}}" /staging && \
+    # Copy resources bundled by SPM to staging area
+    find -L "$(swift build --package-path /build -c release --show-bin-path)/" -regex '.*\.resources$' -exec cp -Ra {} /staging \;
```

The `--package-path /build` options in these lines are not necessary with it set up this way.
```suggestion
    cp "$(swift build -c release --show-bin-path)/{{name}}" /staging && \
    # Copy resources bundled by SPM to staging area
    find -L "$(swift build -c release --show-bin-path)/" -regex '.*\.resources$' -exec cp -Ra {} /staging \;
```